### ISSUE TITLE
Linux build doesn't work

### ISSUE THREAD

**Issue by faveoled** [chidiwilliams/buzz#21]
![image](https://user-images.githubusercontent.com/85760289/193769382-0c8c9767-e519-4d7e-b8bf-c04029606f6a.png)
Linux binary archive contains libraries and executables for macOS, not Linux

**Comment by chidiwilliams** [comment]
Good catch, thanks. How about this? https://github.com/chidiwilliams/buzz/releases/tag/v0.2.3

**Comment by faveoled** [comment]
It's now Linux but responds with Segmentation fault on startup

**Comment by faveoled** [comment]
The archive contains a lot of system libraries, which is not good. The app started running after I manually deleted the following (albeit with some warnings):
- libXss.so.1
- libXrender.so.1
- libXrandr.so.2
- libxkbcommon.so.0
- libXinerama.so.1
- libXi.so.6
- libXft.so.2
- libXfixes.so.3
- libXext.so.6
- libXdmcp.so.6
- libXdamage.so.1
- libXcursor.so.1
- libXcomposite.so.1
- libxcb-xfixes.so.0
- libxcb-sync.so.1
- libxcb-shm.so.0
- libxcb-render.so.0
- libxcb-glx.so.0
- libXau.so.6
- libX11-xcb.so.1
- libX11.so.6

**Comment by faveoled** [comment]
Encountered some error:
```
[2022-10-05 10:44:12,226] transcriber.start_recording:44 DEBUG -> Recording... language: "en", model: "tiny", task: "Task.TRANSCRIBE", device: "5", block duration: "10", sample rate: "44100.0"
Traceback (most recent call last):
  File "gui.py", line 291, in on_status_changed
  File "gui.py", line 317, in start_recording
  File "gui.py", line 206, in start_recording
  File "transcriber.py", line 46, in start_recording
  File "sounddevice.py", line 1421, in __init__
  File "sounddevice.py", line 898, in __init__
TypeError: an integer is required
Aborted (core dumped)
```

**Comment by faveoled** [comment]
By the way the transcription works with this build ("pulse" microphone). I can finally use this program!

**Comment by chidiwilliams** [comment]
> Encountered some error:
> 
> ```
> [2022-10-05 10:44:12,226] transcriber.start_recording:44 DEBUG -> Recording... language: "en", model: "tiny", task: "Task.TRANSCRIBE", device: "5", block duration: "10", sample rate: "44100.0"
> Traceback (most recent call last):
>   File "gui.py", line 291, in on_status_changed
>   File "gui.py", line 317, in start_recording
>   File "gui.py", line 206, in start_recording
>   File "transcriber.py", line 46, in start_recording
>   File "sounddevice.py", line 1421, in __init__
>   File "sounddevice.py", line 898, in __init__
> TypeError: an integer is required
> Aborted (core dumped)
> ```

This should have been fixed in a later build.

**Comment by chidiwilliams** [comment]
> By the way the transcription works with this build ("pulse" microphone). I can finally use this program!

Oh, great to hear! I'll look into the system libraries issue, although I can't say I fully understand it or how to fix it at the moment. Thanks for all your feedback!

**Comment by faveoled** [comment]
> This should have been fixed in a later build.

You mean later than the latest available? I'm using 0.2.3 you referenced

**Comment by chidiwilliams** [comment]
> > This should have been fixed in a later build.
> 
> You mean later than the latest available? I'm using 0.2.3 you referenced

I mean on 0.2.3. You mentioned you were able to get transcription with the "pulse" microphone.

**Comment by faveoled** [comment]
Yeah, the integer error is from HDA Intel, not pulse
